modules/configuring-vsphere-monitoring-configuration-completion.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/installing-vsphere-post-installation-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-vSphere-monitoring-configuration-completion{context}"]
+= Verifying vSphere configuration completion
+
+The connection configuration process updates operator statuses and control plane nodes. It takes approximately an hour to complete. At the end of the configuration process, the nodes will reboot. Previously bound `PersistentVolumeClaims` objects might become disconnected as a result of the configuration updates. 
+
+.Prequisites
+* Save the configuration settings in the *vSphere connection configuration* wizard.
+
+.Procedure
+
+. Check that the configuration process completed successfully:
++
+--
+.. In the OpenShift Container Platform Administrator perspective, navigate to *Home > Overview*.
+.. Under *Status* click *Operators*. Wait for all operator statuses to change from  *Progressing* to *All succeeded*.  A *Failed* status indicates that the configuration failed.
+.. Under *Status*, click *Control Plane*. Wait for the response rate of all Control Pane components to return to 100%. A *Failed* control plane component indicates that the configuration failed.
+--
++
+A failure indicates that at least one of the connection settings is incorrect. Change the settings in the *vSphere connection configuration* wizard and save the configuration again.
+
+
+. Check that the `PersistentVolumeClaims` objects have not been disconnected:
++
+.. Create a `StorageClass` object.
```

Comment:
We might provide hints what values a `StorageClass` needs to contain to use the just configured connection.

See `Diagnostic Steps` in the KB: https://access.redhat.com/solutions/6677901

Reply:
Let's decide on this together. Either I can just say 'using the following parameters' or I can write the GUI procedures out like in the Google Docs:
https://docs.google.com/document/d/1ES2Of_f584bO_D6o3msBYr-9kybEh3bVrnI4X-ZZ8s4/edit#